### DISCUSSION TITLE
I want to know how to check patch version in ofbiz.

### DISCUSSION THREAD

**Discussion by suiiim** [community/community#53023] [Programming Help]
### Select Topic Area

Question

### Body

I can check ofbiz version using VERSION file.
"https://github.com/apache/ofbiz-framework/blob/release18.12.05/VERSION"
But There's only version major and minor version like 18.12.
There's no patch version!

So what can I do checking patch version like 18.12.05.

**Comment by airtower-luna** [comment]
The URL you mentioned contains a release tag, which contains a patch version. Is there any reason you can't just extract the version from the tag instead of downloading the file? :thinking: 

In general, that seems like a question you'd better ask the ofbiz developers (or maybe other users). Here in the general GitHub community the chance to find someone who knows their versioning scheme is probably low. :sweat_smile:
